source/common/network/utility.h

```diff
@@ -80,16 +80,17 @@ class Utility {
   static uint32_t portFromTcpUrl(const std::string& url);
 
   /*
-  * Parse an internet host address (IPv4 or IPv6) and create an Instance from it. Throw
-  * an exception if unable to parse the address. The address must not include a port number.
+  * Parse an internet host address (IPv4 or IPv6) and create an Instance from it. The address
```

Comment:
nit: can you fix the formatting of these comments and for the line above replace /* with /**